src/resources.html

```diff
@@ -0,0 +1,334 @@
+---
+layout: layouts/default
+title: Resources | floe
```

Comment:
Please remove `| floe` from the title in this file, that should be added in the template since every page has this in its title

Reply:
fixed